### ISSUE TITLE
Joule Extension - Failed to connect to node

### ISSUE THREAD

**Issue by Jonbros01** [PierreRochard/node-launcher#93]
Any idea why Joule cant connect to the LN node URL?
<img width="695" alt="screen shot 2018-12-26 at 1 00 09 pm" src="https://user-images.githubusercontent.com/31890680/50453432-85c54180-090e-11e9-87ee-4ef1b69d0ff6.png">

**Comment by PierreRochard** [comment]
Did you follow https://medium.com/lightning-power-users/bitcoin-lightning-joule-chrome-extension-ac149bb05cb9 ?

**Comment by Jonbros01** [comment]
Yes but for some reasons,  if I click on the “clickling this link”  I don’t get to the same Your connection is not private page.  What I get instead is This site can’t be reached, local host refused to connect and there’s no Advanced button to click on.

[cid:image001.png@01D49DD2.D05D3790]

**Comment by Jonbros01** [comment]
I rebooted my iMAC and now it work.
